tck/.dir-locals.el

```diff
@@ -0,0 +1,10 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
+(
+ (dired-mode . ((c-basic-offset . 4)))
+ (java-mode  . ((c-basic-offset . 4)))
+)
+
+
```

Comment:
Please delete this file from source control (and replace it with a local copy after committing if you want) to prevent it from getting merged to master.